aom/src/main/java/com/nedap/archie/aom/utils/NodeIdUtil.java

```diff
@@ -29,7 +29,7 @@ public boolean isRedefined() {
         }
         if(codes.size() > 1) {
             for(int i = 0; i < codes.size()-1;i++) {
-                if(codes.get(i) > 0) {
+                if(Integer.parseInt(codes.get(i)) > 0) {
```

Comment:
This should be a strict check on the string `"0"`, as `at0000.1` is redefined, but `at0.1` is not.

```suggestion
                if(!"0".equals(codes.get(i))) {
```